libs/Lambda2Js/project.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": "1.0.0",
```

Comment:
Match the version with the one that you are using, but increase by some number so it would not take the package from NuGet e.g. 2.0.99